packages/card-picker/tests/acceptance/close-button-test.js

```diff
@@ -0,0 +1,29 @@
+import { module, test } from 'qunit';
+import { visit, click, waitFor, waitUntil, find} from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+
+module('Acceptance | close button', function(hooks) {
+  setupApplicationTest(hooks);
```

Comment:
any particular reason for using an application test over a rendering/integration test?